src/routes/brands/updateBrand.ts

```diff
@@ -25,14 +27,13 @@ const handler: RouteHandler<UpdateBrandRoute> = async (req, reply) => {
     const brandId = req.params.id;
     const newBrandName = req.body.name;
```

Comment:
Destructuring name with an alias...
```suggestion
const {name:newBrandName} = req.body;
```